client/src/components/Maps.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import {
+ GoogleMap, withScriptjs, withGoogleMap, Marker, InfoWindow,
+} from 'react-google-maps';
+import { businesses } from '../constant';
+import classes from '../modules/maps.module.css';
+
+
+function Map() {
+    const [selectedBusiness, setSelectedBusiness] = useState(null);
+    const Markers = () => (
+        <div>
+            {businesses.map(business => (business.needsHelp
```

Comment:
Could use [this](http://es6-features.org/#ObjectMatchingShorthandNotation)
i.e.
```javascript
{businesses.map(({ id, lat, lng, needsHelp }) => (needsHelp
```

Reply:
I am having some issues with changing to object destructuring for 2 reasons.

**Reason 1**:
business to  {id, lat, lng, needsHelp } => only gives me access to those 4 fields. However, down in InfoWindows, I also require access to storeName, street address and avatar. This would mean that object destructuring would be 7 fields. 

**Reason 2**
Lint is asking that I reorganize the new object destructured function into something that is more difficult to read. 

If okay, I would like to keep this the same. However, if you feel strongly about this, please let me know and I will try to change.

Reply:
As I said earlier in the Slack, let's not worry about it 👍 Thanks for the heads up!